easybuild/easyconfigs/m/MUMPS/MUMPS-5.2.1_shared-pord.patch

```diff
@@ -0,0 +1,79 @@
+source: https://src.fedoraproject.org/rpms/MUMPS/blob/master/f/MUMPS-shared-pord.patch
```

Comment:
@lexming please add Author and update the checksum?

Reply:
Done :+1: